src/FSharpVSPowerTools/Commands/PrintfSpecifiersUsageTaggerProvider.cs

```diff
@@ -41,11 +41,12 @@ public class PrintfSpecifiersUsageTaggerProvider : IViewTaggerProvider, IDisposa
             _printfColorManager = printfColorManager;
 
             VSColorTheme.ThemeChanged += UpdateTheme;
+            _printfColorManager.UpdateColors(true);
```

Comment:
Could you use named parameters e.g. `force: true` to make boolean arguments clear?

This comment applies to other invocations of `UpdateColors` as well.